lib/truffle/java/java_utilities.rb

```diff
@@ -224,6 +224,7 @@ def self.make_proxy(a_class)
             Java.invoke_java_method(CLASS_GET_SIMPLE_NAME, a_class)),
           a_proxy) unless parent == nil
       rescue
+        nil
```

Comment:
It's sad this style does not enforce a better handling.
I think a better handling is `warn e.message` and then even better find the root cause of this anti-pattern.
Could you TODOs to those without a justification?